cfme/tests/automate/test_service_automate.py

```diff
@@ -397,4 +400,10 @@ def test_import_dialog_file_without_selecting_file():
             1. Flash message: "At least 1 item must be selected for export"
             2. Error flash message should not appear
     """
-    pass
+    import_export = DialogImportExport(appliance)
+    view = navigate_to(import_export, "DialogImportExport")
+    view.export.click()
+    view.flash.assert_message("At least 1 item must be selected for export")
+    if BZ(1740796).blocks:
+        with pytest.raises(AssertionError, match="At least 1 item must be selected for export"):
```

Comment:
This will only check the flash message till the BZ is fixed right ? 
If you are automating this BZ it should be tested even after the BZ is verified so you can block the test until the BZ is fixed and once the BZ is fixed it should check that the flash message is not shown anymore .
What do you think ?

Reply:
Yeah I would say that this test shouldn't run until the BZ is fixed, because this is the main assertion of the test. The bz should just be in the `blockers` metadata

Reply:
@john-dupuy 
- According to me, we don't need to skip this entire test case. Although clicking on export button without selecting dialog gives expected error flash message.
@sshveta 
- I agree with you. I changed test case like below:
```
 if not BZ(1740796).blocks:
        dialog.update({'label': fauxfactory.gen_alphanumeric()})
```
Please le me know, if we need any other changes.

Reply:
@ganeshhubale While this solution is good once the BZ is fixed, right now the test case is passing. It should be failing because the BZ is still present. In my view, a test case for a specific BZ shouldn't pass until the BZ is fixed. This is why I think it is better to block the whole test case.

Reply:
@john-dupuy 
- I agree that this BZ is not yet fixed. But this BZ is only related to this issue - ```Error flash message after updating dialog```
- So I am skipping execution of this part until this BZ got fixed. Once it fixed, it will check if there is error flash message after updating dialog.
- Please let me know your view.

Reply:
Chatted offline, eventually settled on blocking the test case with `blockers`